server/hotrod/src/main/java/org/infinispan/server/hotrod/MetadataUtils.java

```diff
@@ -0,0 +1,49 @@
+package org.infinispan.server.hotrod;
+
+import org.infinispan.container.entries.CacheEntry;
+import org.infinispan.container.versioning.EntryVersion;
+import org.infinispan.container.versioning.NumericVersion;
+import org.infinispan.container.versioning.SimpleClusteredVersion;
+
+public class MetadataUtils {
```

Comment:
@rvansa I created this class to handle the metadata directly from the entry instead calculating things in the other method